templates/emails/email_base.html

```diff
@@ -0,0 +1,42 @@
+{% load static %}
+{% load inlinecss %}
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="Content-Type" content="text/html charset=UTF-8"/>
+    <title>Confirm your email address</title>
+    <style>
+        @import url('https://fonts.googleapis.com/css?family=Chivo:400,400i,700|Playfair+Display:400,700&display=swap');
+    </style>
+</head>
+{% inlinecss "css/emails.css" %}
+    <body>
+    <div class="main">
+        <img class="opencraft-logo" src="{{ base_url }}{% static 'img/svg/logo-for-email.svg' %}"
+             alt="Opencraft Logo" aria-hidden="true">
+        <div class="main">
+            <img src="{{ base_url }}{% static 'img/jpg/header-background.jpg' %}" alt="Email Header" aria-hidden="true">
+            <div class="content">
```

Comment:
![layout](https://user-images.githubusercontent.com/3189670/70830672-0c5d8200-1df1-11ea-9bed-b690db3ef170.png)

@xitij2000, there is a weird padding between header and content and the header is not scaled correctly.

Reply:
This is odd. I was not getting this while testing, but then again I moved on from verify to other emails so perhaps I broke the verify emails at some point.

Reply:
Found the issue! Linking to an asset from localhost doesn't work on gmail so I had uploaded these assets to an s3 bucket of mine and linked from there temporarily. When switching back to the regular image I forgot to copy over the class I'd applied.

Reply:
Ah, got it. Looks great now!